src/goto-harness/goto_harness_parse_options.cpp

```diff
@@ -174,7 +174,8 @@ void goto_harness_parse_optionst::help()
     << " goto-harness <in> <out> --harness-function-name <name> --harness-type "
        "<harness-type> [harness options]\n"
     << "\n"
-    << "<in> <out>                 goto binaries to read from / write to\n"
+    << "<in>                       goto binaries to read from / write to\n"
```

Comment:
Should this say "goto binary to read from"?

Reply:
:facepalm: